pages/shared.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from "react";
+import Banner from "../components/domains/shared/Banner";
+import CardList from "../components/commons/CardList";
+import SearchInput from "../components/commons/SearchInput";
+import styles from "../styles/sharedPage.module.css";
+import { getSharedData } from "./api/SharedApi";
+import { Folder } from "../types/shared";
+import { Link } from "../types/common";
+
+function SharedPage({ openModal }: { openModal: () => void }) {
+  const [searchResult, setSearchResult] = useState<string>("");
+  const [sharedFolder, setSharedFolder] = useState<Folder>({
+    id: 1,
+    name: "",
+    owner: {
+      id: 2,
+      name: "",
+      profileImageSource: "",
+    },
+    links: [
+      {
+        id: 1,
+        url: "",
+        title: "",
+        description: "",
+        image_source: "",
+        created_at: "",
+      },
+    ],
+  });
+
+  const handleFolderLoad = async () => {
+    const { folder } = await getSharedData();
+    setSharedFolder({
+      ...folder,
+      links: folder.links,
+    });
+  };
+
+  const searchLink = async (keyword: string) => {
```

Comment:
키워드는 이미 위에서 알고있는거같아요
```suggestion
  const searchLink = async () => {
```